File: Book.java
public class Book {
    private String  id;
    private String title;
    private String author;
    private int year;
    private boolean itsAvailable = true;

    public Book(String id, String title, String author, int year, boolean itsAvailable) {
        this.id = id;
        this.title = title;
        this.author = author;
        this.year = year;
        this.itsAvailable = itsAvailable;
    }
    public int getYear() {
        return year;
    }

    public String getId() {
        return id;
    }

    public String getTitle() {
        return title;
    }
    public String getAuthor() {
        return author;
    }

    public boolean isItsAvailable() {
        return itsAvailable;
    }

    public void setItsAvailable(boolean itsAvailable) {
        this.itsAvailable = itsAvailable;
    }
}

File: book.py
class Book:
    def __init__(self, id, title, author, year, its_available=True):
        self.id = id
        self.title = title
        self.author = author
        self.year = year
        self.its_available = its_available

    def get_year(self):
        return self.year

    def get_id(self):
        return self.id

    def get_title(self):
        return self.title

    def get_author(self):
        return self.author

    def is_its_available(self):
        return self.its_available

    def set_its_available(self, its_available):
        self.its_available = its_available
